pkgs/development/cuda-modules/packages/nccl.nix

```diff
@@ -114,10 +116,17 @@ backendStdenv.mkDerivation (finalAttrs: {
 
   enableParallelBuilding = true;
 
-  postFixup = ''
-    _overrideFirst outputStatic "static" "lib" "out"
-    moveToOutput lib/libnccl_static.a "''${!outputStatic:?}"
-  '';
+  postFixup =
+    # Since CUDA 12.8, the cuda_nvcc path leaks in the libnccl.so's `.nv_fatbin` section:
```

Comment:
Let's link https://github.com/NixOS/nixpkgs/pull/457424?notifications_query=is%3Aunread#issuecomment-3477820902 and the commit id?